src/Bicep.Core.Samples/Files/InvalidTypeDeclarations_LF/main.formatted.bicep

```diff
@@ -9,6 +9,11 @@ type resource = bool
 @sealed()
 type sealedString = string
 
+@sealed()
+type sealedDictionary = {
+  *:string
```

Comment:
Looks like this may need some changes to the formatter to support a space after the `:`